torch/nn/modules/loss.py

```diff
@@ -117,7 +117,7 @@ class NLLLoss(_WeightedLoss):
         l_n = - w_{y_n} x_{n,y_n}, \quad
         w_{c} = \text{weight}[c] \cdot \mathbb{1}\{c \not= \text{ignore\_index}\},
 
-    where :math:`N` is the batch size. If :attr:`reduction` is not ``'none'``
+    where :math:`N` is the batch size and :math:`y_n` is the value of target at index n. If :attr:`reduction` is not ``'none'``
```

Comment:
Use `` around n ("at index n").

Github seems to be dropping my comments, so I am adding this one again.